lib/options/uefi-setup-menu.robot

```diff
@@ -143,3 +143,20 @@ Measure Reboot Time
     ${min}    ${max}    ${average}    ${stddev}=
     ...    Calculate Boot Time Statistics    ${durations}
     RETURN    ${min}    ${max}    ${average}    ${stddev}
+
+Make Sure That Flash Locks Are Disabled
+    [Documentation]    Keyword makes sure firmware flashing is not prevented by
+    ...    any Dasharo Security Options, if they are present.
+    IF    not ${DASHARO_SECURITY_MENU_SUPPORT}    RETURN
+    Log To Console    Make Sure That Flash Locks Are Disabled
+    Power On
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    Get Flashrom From Cloud
+    ${out_flashrom}=    Execute Command In Terminal    flashrom -p internal
+    # Should Not Contain    ${out_flashrom}    read-only
+
+Login To Windows
```

Comment:
I am unsure how it helps. 

At best it triggers conflict:

```
Multiple keywords with name 'Login To Windows' found. Give the full name of the keyword you want to use:
    keywords.Login To Windows
    uefi-setup-menu.Login To Windows
```

Reply:
Removed in: f518bda2fe7846ef332062f00ef98b830ad5698a